requirements-dev.txt

```diff
@@ -5,7 +5,10 @@ treon>=0.1.3
 pytest>=6.2.5
 mypy>=0.780
 mypy-extensions>=0.4.3
-
+jupyter-sphinx==0.3.2
+nbsphinx==0.8.8
+sphinx-autodoc-typehints==1.17.0
+reno==3.5.0
```

Comment:
We might want to carefully consider whether to use the `==`, `~=`, or `>=` [specifier](https://www.python.org/dev/peps/pep-0440/#version-specifiers) for each of these.